### ISSUE TITLE
3rd party libs

### ISSUE THREAD

**Issue by trusktr** [raix/Meteor-famono#99]
@raix You're `library` for importing 3rd party libs is awesome. I think it might be nice if you made this a separate package, f.e. `raix:library` without any default libs. And it can be a proof of concept for Meteor to possibly adopt or make something like it.

@dandv ^ Check out raix:famono on Atmospherejs and see how it loads non-Atmosphere libraries for your project.

**Comment by trusktr** [comment]
@raix Oh, is that what raix:include is for?

**Comment by raix** [comment]
Yep, raix:include is just a tiny helper package for the meteor bower/github release i did. It's a proof of concept - the Nice thing about that approach is the use of the version resolver on 3rd party packages.
The include package should/could be removed by adding scope detection and isolation (we can do alot of thing here)
